srcpkgs/vagrant/template

```diff
@@ -25,6 +26,16 @@ do_build() {
 do_install() {
 	GEM_PATH=$DESTDIR/usr/lib/$pkgname GEM_HOME="$GEM_PATH" gem install $pkgname-$version.gem --no-document -- --disable-clean
 
+	vlicense LICENSE
+
+	vmkdir usr/bin
+	vbin $FILESDIR/$pkgname
+
+	vcompletion "contrib/bash/completion.sh" bash
+
+	mkdir -p /usr/share/vim/vim90/ftplugin/
+	cp contrib/vim/vagrantfile.vim /usr/share/vim/vim90/ftplugin/
```

Comment:
use vcopy or vinstall, (vinstall should get rid of the mkdir too, which is broken anyways... you'll probably want to `./xbps-src zap && ./xbps-src binary-bootstrap`, you're polluting the masterdir)

Reply:
Thanks for the advice. I've changed it accordingly.